cli/internal/upgrade/iammigrate.go

```diff
@@ -1,110 +0,0 @@
-/*
```

Comment:
nit: could you move the file next time instead of copying and deleting it? makes the git diff easier to read :)

Reply:
afaik Git marks files automatically as "moved" if create a new file with content from a deleted file, and the new file isn't too different from the old one.
This isn't the case here since most of the code was removed/rewritten.
If this is something bothering you a lot, I can try to keep the moving and code changes to the file in two separate commits next time, to make it clearer if a file was "moved".

Reply:
no it's fine :) didn't know that the detection is automatic. I just realized that it was shown as moved in another PR.